SmartTutor/Controllers/ChallengeProjectController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using SmartTutor.ActiveEducationModel;
+using SmartTutor.Repository.ChallengeProjectRepository;
+using SmartTutor.Service;
+
+namespace SmartTutor.Controllers
+{
+    public class ChallengeProjectController
```

Comment:
We should refactor this to reflect the new design. I think our new view simplifies the overall design a lot, so examine the classes and think about the objects that will be instantiated (and how they will interact) when a request from the IDE arrives.